docs/config.rst

```diff
@@ -41,3 +41,20 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
     :default: ``https://${OIDC_DOMAIN}/userinfo``
 
    The audience for the access token that is generated by Auth0.
+
+
+.. envvar:: REACT_APP_REFRESH_AUTH_PERIOD_SECONDS
+
+    :default: 300
+
+    How often to issue a silent authentication refresh. Technically, when you're logged in,
```

Comment:
It's not how often to issue a silent authentication refresh, but how often do we check if the expiry is in less than `REACT_APP_REFRESH_AUTH_PREEMPTIVE_SECONDS`.